apps/gitness/src/components/LinkComponent.tsx

```diff
@@ -0,0 +1,7 @@
+import { Link } from 'react-router-dom'
+
+import { TLinkComponent } from '@harnessio/ui/views'
+
+export const LinkComponent: TLinkComponent = ({ to, children }: { to: string; children: React.ReactNode }) => (
```

Comment:
not blocking, but let's add a disabled prop to this component to have pointer events as none.